src/main/java/seedu/workbook/model/VersionedWorkBook.java

```diff
@@ -92,6 +95,10 @@ public boolean equals(Object other) {
                 && currentStatePointer == otherVersionedAddressBook.currentStatePointer;
     }
 
+    public boolean internshipHasNoTips(Internship internship) {
```

Comment:
why is there a need to define this method here when its already in `WorkBook.java`?